lib/groonga-query-log/server-verifier.rb

```diff
@@ -33,12 +33,13 @@ def initialize(options)
 
     def verify(input, &callback)
       @same = true
+      @client_error = false
```

Comment:
`@client_error` will have error object.
`@client_error_is_occurred` or something is better.